Demo.Api/Startup.cs

```diff
@@ -28,6 +28,13 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             // service dependencies
+            // Using Scrutor to map the dependencies with scoped lifetime (https://github.com/khellang/Scrutor)
+            services.Scan(scan => scan
```

Comment:
Please move this Scrutor declaration in Service Extension class. Goal is to keep the ConfigureServices method clean and readable as much as possible as a part of best practices. 

[https://code-maze.com/aspnetcore-webapi-best-practices/#startupclass](url)

Reply:
Sounds good. Updated!